src/ComputerPlayer.java

```diff
@@ -4,6 +4,9 @@ public class ComputerPlayer extends Player{
 	
 	public ComputerPlayer(){
 		super("R2D2");
+//		The computer never assigned the name
+//		to be 'R2D2' so it was only null when running
+		myName = "R2D2";
```

Comment:
This only fixes the name for the ComputerPlayer class.  Since it passed the name to the Player class, it's better to make the fix there.